manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $manage_squid = true
```

Comment:
i think we want this to be false by default, and turn it on here: 
https://github.com/Katello/puppet-katello/blob/master/manifests/init.pp#L96-L119
and 
https://github.com/Katello/puppet-capsule/blob/master/manifests/init.pp#L163-L187
and
https://github.com/Katello/puppet-katello_devel/blob/master/manifests/init.pp#L126-L144

and yes there has been discussion on consolidating these parts :)